<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(9,36,43,255)</string>
  </property>
  <widget class="QLabel" name="name_Label2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SCANNING</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Downloads/icons8-stan-marsh-96.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_label1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);</string>
   </property>
   <property name="text">
    <string>VULNERABILITY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="target_LineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgba(80,218,107,255);
color: rgb(255, 255, 255);
text-align: center;
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter a URL, IP address or hostname</string>
   </property>
  </widget>
  <widget class="QLabel" name="target_Label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> TARGET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stealth_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>110</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(155,233,172,255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string>Stealth </string>
   </property>
  </widget>
  <widget class="QPushButton" name="aggressive_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>110</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(155,233,172,255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string> Aggressive </string>
   </property>
  </widget>
  <widget class="QPushButton" name="adaptive_Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>110</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(155,233,172,255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string>Adaptive </string>
   </property>
  </widget>
  <widget class="QLabel" name="output_Label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>511</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(27,79,96,70%);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;
color: rgb(255, 255, 255);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
border-color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="cve_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>611</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(27,79,96,70%);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bg_date_Label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(27,79,96,70%);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="send_Button">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>640</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;


</string>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_mail">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>640</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgba(80,218,107,255);
color: rgb(255, 255, 255);
text-align: center;
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter e-mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="send_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>640</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SEND FULL REPORT TO E-MAIL</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_Label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>Date-Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_date_Label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>130</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="vulner_Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>110</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(155,233,172,255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string>Vulner.NSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scan_Button">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;


</string>
   </property>
   <property name="text">
    <string>SCAN</string>
   </property>
  </widget>
  <widget class="QLabel" name="host_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>551</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="oyrput_OS_Label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="options_LineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgba(80,218,107,255);
color: rgb(255, 255, 255);
text-align: center;
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Options</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="result_tabWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>200</y>
     <width>491</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: #f5f5f5;
alternate-background-color: #e0e0e0;
selection-background-color: #d7d7d7;
selection-color: #000000;
border: none;


</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_Button">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;


</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="statusScan_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="date_Label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(240,50,57,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
